package.json

```diff
@@ -1,9 +1,13 @@
 {
-  "repository": "GoogleCloudPlatform/google-cloud-node",
   "name": "redis",
+  "description": "Google Cloud Memorystore for Redis API client for Node.js",
   "version": "0.1.0",
+  "license": "Apache-2.0",
   "author": "Google LLC",
-  "description": "Google Cloud Memorystore for Redis API client for Node.js",
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "repository": "@google0cloud/redis",
```

Comment:
s/0/-/

Reply:
Actually, that's wrong. `repository` should be `"googleapis/nodejs-redis"` (it's a GitHub location), and `name` should be `"@google-cloud/redis"`.